File: c99/bc1858.c
#include<stdio.h>
#include<string.h>

int main()
{
	int nPe, menor, mIn = 1, val, i, j = 0;
	
	scanf("%d", &nPe);
	int cVal[nPe];
	char sIn[nPe*2];
	
	fflush(stdin);
	
	gets(sIn);
	
	for(i = 0; i < nPe*2; i++)
	{
		if(sIn[i] != ' ')
		{
			cVal[j] = sIn[i] - '0';
			j++;
		}
	}
	
	menor = cVal[0];
	
	for(i = 1;i < nPe; i++)
	{
		if(cVal[i] < menor)
		{
			menor = cVal[i];
			mIn = i + 1;
		}
	}
	
	printf("%d\n", mIn);
}

File: python/bc1858.py
mIn = 1

nPe = int(input())
text = input().split(" ")

cVal = [int for j in range(nPe)]

i = 0
for c in range(0, len(text)):
    if text[c] != " ":
        cVal[i] = text[c]
        i += 1

menor = cVal[0]

for j in range(1, nPe):
    if cVal[j] < menor:
        menor = cVal[j]
        mIn = j + 1

print(mIn)